packages/editor/src/client/panel/embedEditPanel.tsx

```diff
@@ -17,14 +17,13 @@ import {EmbedBlockValue, EmbedType} from '../blocks/types'
 
 export interface EmbedEditPanel {
   readonly value: EmbedBlockValue
-
   onClose(): void
   onConfirm(value: EmbedBlockValue): void
 }
 
 export function EmbedEditPanel({value, onClose, onConfirm}: EmbedEditPanel) {
   const [errorMessage, setErrorMessage] = useState<string>()
-
+  const [ratio, setRatio] = useState<boolean>(true)
```

Comment:
shouldn't it get the start value from `value`?

Reply:
ideally yes, but this is listening to the input and changes, a state was more reliable due i had some irregular behaviour with the value

Reply:
ratio was droped